api/migrations/20180420140208_other-funding-sources.js

```diff
@@ -0,0 +1,7 @@
+exports.up = async knex =>
+  knex.schema.alterTable('activities', table => {
+    table.text('other_funding_sources_description');
+    table.decimal('other_funding_sources_amount', 12, 2);
+  });
+
+exports.down = async () => {};
```

Comment:
It turns out you can't rollback individual migrations with knex - you either rollback everything or nothing.  As a result, there's no point in having `down` migrations unless the `up` creates a table or key constraint.  :shrug:

Reply:
huh, that is strange